src/topology/algebra/weak_dual_topology.lean

```diff
@@ -0,0 +1,87 @@
+/-
+Copyright (c) 2021 Kalle Kytölä. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Kalle Kytölä and Heather Macbeth
```

Comment:
I'm flattered, but this is your work alone!

Reply:
I don't think that's exactly accurate (almost all of the good quality code is based on your input here and on Zulip, and the whole idea of defining weak duals in this way was yours). But whichever way you prefer is of course ok.

Reply:
I hope we will write something together some time, but this is really all your work :)

Reply:
Ok, I removed your name from "Authors". It would indeed be really nice to write something together, but I maintain that I did not write this one without you either :smile:.